carma-platform.repos

```diff
@@ -1,48 +1,60 @@
 repositories:
+  carma/src/astuff_sensor_msgs:
+    type: git
+    url: https://github.com/astuff/astuff_sensor_msgs.git
+    version: melodic
+  autoware.ai:
+    type: git
+    url: https://github.com/mjeronimo/autoware.ai.git
```

Comment:
Well want to point this at our autoware.ai fork once that is merged. Same with the other links in this file

Reply:
Yes, I've added a tasks to make sure this doesn't get missed. We can integrate carma-base and autoware.ai first and then do this PR last, updating the URLs before merging.